src/platform/packages/shared/kbn-unified-tabs/src/types.ts

```diff
@@ -26,6 +26,25 @@ export interface TabsSizeConfig {
   // TODO: extend with possibly different sizes for pinned tabs
 }
 
+// TODO adjust interface when real data is available, this currently types TAB_CONTENT_MOCK
+export interface PreviewContentConfig {
+  id: number;
+  name: string;
```

Comment:
Is it the same as `item.id` and `item.label`? We could pass `item` as a separate prop if this data is required in the preview.

Reply:
I didn't think through this config yet, as we don't know the shape of the data. It was just to provide mocked items I need for UI, but all namings and content can and will be adjusted.

I can pass the whole item as a separated prop, no problem! I wanted to keep all preview content stuff in one prop, but I don't have strong opinion about that

Reply:
Separated preview query and preview item in 2 props